pkg/scheduler/apis/config/types_test.go

```diff
@@ -32,38 +32,38 @@ func TestPluginsAppend(t *testing.T) {
 		{
 			name: "AppendPlugin",
 			customPlugins: &Plugins{
-				Filter: &PluginSet{
+				Filter: PluginSet{
 					Enabled: []Plugin{
 						{Name: "CustomPlugin"},
 					},
 				},
 			},
 			defaultPlugins: &Plugins{
-				Filter: &PluginSet{
+				Filter: PluginSet{
 					Enabled: []Plugin{
 						{Name: "DefaultPlugin1"},
 						{Name: "DefaultPlugin2"},
 					},
 				},
 			},
 			expectedPlugins: &Plugins{
-				QueueSort: &PluginSet{},
-				PreFilter: &PluginSet{},
-				Filter: &PluginSet{
+				QueueSort: PluginSet{},
+				PreFilter: PluginSet{},
+				Filter: PluginSet{
 					Enabled: []Plugin{
 						{Name: "DefaultPlugin1"},
 						{Name: "DefaultPlugin2"},
 						{Name: "CustomPlugin"},
 					},
 				},
-				PostFilter: &PluginSet{},
-				PreScore:   &PluginSet{},
-				Score:      &PluginSet{},
-				Reserve:    &PluginSet{},
-				Permit:     &PluginSet{},
-				PreBind:    &PluginSet{},
-				Bind:       &PluginSet{},
-				PostBind:   &PluginSet{},
+				PostFilter: PluginSet{},
```

Comment:
so now you can actually remove these lines that have empty PluginSet :)

Reply:
Removed